### PR TITLE
Updated dangerous areas doc

### PR DESCRIPTION
**Pull request by jiedongding**
Initial dangerous areas doc .

### CODE REVIEW

File: DangerousAreas.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Dangerous Areas of Dublin
+
+* Crumlin
```
Comment (darth30joker) [inline DangerousAreas.md]: Can we break them into different areas? like dublin 1, dublin 2, etc. Thanks!

**Review by darth30joker — CHANGES_REQUESTED** [review]

File: DangerousAreas.md (hunk 1 above)
Comment (jiedongding) [reply DangerousAreas.md]: I don't think it's necessary? As there aren't that many areas for each district. And some of the area are in the middle of two districts. Could break them into cities/counties in the future though.

Comment (jiedongding) [reply DangerousAreas.md]: Maybe I can do: south Dublin city, north Dublin city, south Dublin county etc.?

Comment (darth30joker) [reply DangerousAreas.md]: then just south Dublin and north Dublin. what do you say?

Comment (jiedongding) [reply DangerousAreas.md]: Just updated, does it look ok?

**Review by darth30joker — APPROVED** [review]

**Comment by darth30joker** [comment]
LGTM. are you able to merge it?

**Comment by jiedongding** [comment]
Doesn't seem to be able to.

**Pull request merged**